x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -3459,7 +3459,6 @@
     "kibana-react.solutionNav.collapsibleLabel": "サイドナビゲーションを折りたたむ",
     "kibana-react.solutionNav.mobileTitleText": "{solutionName}メニュー",
     "kibana-react.solutionNav.openLabel": "サイドナビゲーションを開く",
-    "kibana-react.splitPanel.adjustPanelSizeAriaLabel": "左右のキーを押してパネルサイズを調整します",
```

Comment:
Could you change the label of the translation to `console.splitPanel.adjustPanelSizeAriaLabel` instead of removing it? Thanks)